projects/plugins/jetpack/extensions/shared/components/stripe-nudge/style.scss

```diff
@@ -15,3 +15,7 @@
 	margin-right: 16px;
 	padding: 6px;
 }
+
+.jetpack-stripe-nudge__banner .jetpack-stripe-nudge__link {
+	color: #0675c4 !important;
```

Comment:
I think the !important should not be needed, right?

Reply:
It is needed.

Reply:
You could add [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) to overcome it and not rely on `!important` which is a bit blunt. A bit of a hack is to do `.jetpack-stripe-nudge__link.jetpack-stripe-nudge__link` or include `+ span` or `span.class`

Reply:
I was asking because I thought you [created a custom class](https://github.com/Automattic/jetpack/pull/30320#discussion_r1179310488)...

Reply:
Thanks for the feedback!

> You could add [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) to overcome it and not rely on !important which is a bit blunt. A bit of a hack is to do .jetpack-stripe-nudge__link.jetpack-stripe-nudge__link

`jetpack-stripe-nudge__link.jetpack-stripe-nudge__link` would not work in this case

> + span or span.class

Umm, is `spain.class` similar to the initial approach above or are you referring to a different `span.css` class? ` + span` would not be specific enough in this case as well.

Made the CSS more specific without `!important. See [a83c758](https://github.com/Automattic/jetpack/pull/30320/commits/a83c758014c6dbe50dce93abfee90a5619b6f785)